kubernetes/services/concent-api-worker.yml.j2

```diff
@@ -0,0 +1,55 @@
+apiVersion: extensions/v1beta1
+kind:       Deployment
+metadata:
+  name: concent-api-worker
+spec:
+  replicas:             3
+  revisionHistoryLimit: 3
+  template:
+    metadata:
+      labels:
+        run: concent-api-worker
+    spec:
+      containers:
+        - name:             concent-api-worker
+          image:            {{ docker_registry }}/{{image_prefix }}concent-api:{{ image_version }}
+          imagePullPolicy:  Always
+          command:          ["/usr/local/bin/run-celery.sh"]
+          ports:
+            - containerPort: 80
+          resources:
+            requests:
+              memory: "0.6Gi"
+              cpu:    "50m"
+            limits:
+              memory: "1Gi"
```

Comment:
These memory limits are pretty high. Can we set them lower?

Reply:
Ok. I change them to the same resources as `conductor`.